db/db_properties_test.cc

```diff
@@ -1175,6 +1175,60 @@ class CountingDeleteTabPropCollectorFactory
   }
 };
 
+class BlockCountingTablePropertiesCollector : public TablePropertiesCollector {
+ public:
+  static const std::string kNumSampledBlocksPropertyName;
+
+  const char* Name() const override {
+    return "BlockCountingTablePropertiesCollector";
+  }
+
+  Status Finish(UserCollectedProperties* properties) override {
+    (*properties)[kNumSampledBlocksPropertyName] =
+        ToString(num_sampled_blocks_);
+    return Status::OK();
+  }
+
+  Status AddUserKey(const Slice& /*user_key*/, const Slice& /*value*/,
+                    EntryType /*type*/, SequenceNumber /*seq*/,
+                    uint64_t /*file_size*/) override {
+    return Status::OK();
+  }
+
+  void BlockAdd(uint64_t /* blockRawBytes */, uint64_t blockCompressedBytesFast,
```

Comment:
Nit: block_compressed_bytes_fast and block_compressed_bytes_slow?
To make the linter happy, we need to update the variable name in `TablePropertiesCollector::BlockAdd()` too.

Reply:
Done.